help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -122,23 +120,33 @@ Once the destination connection has been created, you can proceed to the audienc
 
 ### Map attributes and identities {#map}
 
-The next step is mapping source identifiers to the Magnite device_id identifier.
+The next step is mapping Source Field identifiers to the [!DNL Magnite] Device Type identifier of your choice, located in the Target Field.
 
-* You can add as many mappings as you need by selecting **[!UICONTROL Add new mapping]**.
+The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.
 
-This example using the Real-Time destination shows a row that contains a generic deviceId source identifier mapped to the Magnite device_id target field. When you're with the mappings, select [!UICONTROL Next].
+In order to support a dynamic list of device types, the dropdown will provide a list of integers (from 0-50), each of which represents a specific device type in [!DNL Magnite]'s system. To understand which device type id in the list of integers corresponds to which [!DNL Magnite] device type, reach out to your [!DNL Magnite] account representative. 
```

Comment:
Multiple issues here:

- he dropdown will provide a list of integers (from 0-50): there is no dropdown in any of the mapping configuration steps. Please rephrase so it reflects the UI.
- Is there some Magnite documentation where these integers are explained? Or is this a custom implementation with no public info? Having to reach out to Magnite whenever a user doesn't know what to select is not a great experience.